src/main/scala/pureconfig/WordDelimiter.scala

```diff
@@ -0,0 +1,42 @@
+package pureconfig
+
+trait WordDelimiter {
+  def toTokens(s: String): Seq[String]
```

Comment:
Sure :). I'm still not sure how the library should handle exceptional behaviour for some fields. This PR is a first attempt to have a configuration but there are many things to decide.